src/bots/userPnlSettler.ts

```diff
@@ -156,21 +210,32 @@ export class UserPnlSettlerBot implements Bot {
 
 	public async startIntervalLoop(intervalMs?: number): Promise<void> {
 		logger.info(`${this.name} Bot started!`);
+		const negativeInterval = intervalMs || this.defaultIntervalMs;
+
 		if (this.runOnce) {
-			await this.trySettlePnl();
+			await this.trySettleNegativePnl();
 			await this.trySettleUsersWithNoPositions();
```

Comment:
should we run the pos. pnl users too on startup?

```suggestion
			await this.trySettleUsersWithNoPositions();
            await this.trySettlePositivePnlForLowMargin();
```